components/sections/CommunitySupportedNumbers.tsx

```diff
@@ -1,9 +1,30 @@
 import { trackEvent } from 'lib/posthog'
-import { DISCORD_URL } from '../../lib/constants'
-
+import { DISCORD_URL, GITHUB_REPO } from '../../lib/constants'
 import Link from 'next/link'
+import { useEffect, useState } from 'react'
 
 const CommunitySupportedNumbers = () => {
+	const [stars, setStars] = useState(null);
+	const githubToken = process.env.GITHUB_ACCESS_TOKEN;
+	useEffect(() => {
+		const apiUrl = `https://api.github.com/repos/${GITHUB_REPO}`;
+		const options = {
+			headers: {
+				Authorization: `token ${githubToken}`,
```

Comment:
FYI, for <https://plabayo.tech/> I get the star count service from the same API. As long as you cache it long enough and given that you only use this one endpoint, you should be fine. You can do up to 60 requests / hour without authentication.

Just saying, as the less API keys you need to keep around and protect, the more secure your posture is by definition. At least that's what i think about it .

Reply:
And I have chosen to preload it at startup myself and only refresh it on success. This way it’s non blocking and always available

Reply:
I don't know how to cache it, do you know a good way of doing that?